js/gated-content/src/views/LiveStreamPage.vue

```diff
@@ -148,6 +148,14 @@ export default {
           // We need here small hack for equipment.
           // In included we have all referenced items, but in relationship only one.
           // So we need manually pass this items to this.video.attributes.equipment.
+          let embedObj = this.video.attributes.media.field_media_video_embed_field;
```

Comment:
1. The comment above is not related to the succeeding code fragment.

2. To simplify the further code support, I advise to not do such data changes on the fly -- but to change the behavior of the `MediaPlayer` itself. Otherwise, we would need the added code fragment each place we use the `MediaPlayer` component.